src/components/Search/SearchByNameResults.module.scss

```diff
@@ -41,3 +41,8 @@
   flex: 1;
   min-width: 0;
 }
+
+.loadingWrapper {
+  // TODO: refactor this: remove important
```

Comment:
Instead of using !important, the selector priority should be increased. Here, we can simply move it under the `.searchResultsPanelWrapper` selector.

Reply:
updated by [47b5932](https://github.com/Magickbase/ckb-explorer-frontend/pull/179/commits/47b59321e07fdf18d81f8f756d11ec5058fe904b)